### ISSUE TITLE
large_tuple incorrectly identifies type annotation as tuple

### ISSUE THREAD

**Issue by ArtisOracle** [realm/SwiftLint#1234]
The rule `large_tuple` indicates the type annotation for a multi-parameter closure is a malformed tuple.

```
open class NetworkSessionHandler: NSObject, URLSessionDataDelegate {
    ...
    var completionHandler: ((_ data: Data?, _ resp: URLResponse?, _ e: NSError?) -> Void)!
```
The type of the variable `completionHandler` is not a tuple but instead a function that accepts 3 arguments and returns `Void` or `()`.

**Comment by marcelofabri** [comment]
This shouldn't trigger. I was able to reproduce it with 0.16.1. Thanks for letting us know!
